platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -55,6 +56,10 @@
 
 	private final GodotRenderView mRenderView;
 	private final InputManagerCompat mInputManager;
+	private final static int MOUSE_WHEEL_UP = 1 << 6; // uses global input indexes
```

Comment:
Instead of "use global input indexes" there should be a comment mentioning `display_server_android.h` since the values need to stay in sync.